### ISSUE TITLE
"Thread Hiding" crashed

### ISSUE THREAD

**Issue by saxamaphone69** [TuxedoTako/4chan-xt#30]
```
"Thread Hiding" crashed on node Post No.76759434 (g). TypeError: Cannot read properties of null (reading 'querySelectorAll')
    at $$ (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?name=4chan-XT.user.js&id=10d5d7a8-76a3-49ab-a7bd-3d6108ba965d:1870:59)
    at Object.makeStub (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?name=4chan-XT.user.js&id=10d5d7a8-76a3-49ab-a7bd-3d6108ba965d:9302:25)
    at Object.hide (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?name=4chan-XT.user.js&id=10d5d7a8-76a3-49ab-a7bd-3d6108ba965d:9361:27)
    at Post.node (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?name=4chan-XT.user.js&id=10d5d7a8-76a3-49ab-a7bd-3d6108ba965d:9182:29)
    at Callbacks.execute (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?name=4chan-XT.user.js&id=10d5d7a8-76a3-49ab-a7bd-3d6108ba965d:372:23)
    at Object.callbackNodes (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?name=4chan-XT.user.js&id=10d5d7a8-76a3-49ab-a7bd-3d6108ba965d:27026:12)
    at Fetcher.fetchedPost (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?name=4chan-XT.user.js&id=10d5d7a8-76a3-49ab-a7bd-3d6108ba965d:7668:14)
    at XMLHttpRequest.<anonymous> (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?name=4chan-XT.user.js&id=10d5d7a8-76a3-49ab-a7bd-3d6108ba965d:7572:23)
    at chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?name=4chan-XT.user.js&id=10d5d7a8-76a3-49ab-a7bd-3d6108ba965d:21190:39
    at execTask (chrome-extension://dhdgffkkebhmkfjojejmpbldmpobfkfo/userscript.html?name=4chan-XT.user.js&id=10d5d7a8-76a3-49ab-a7bd-3d6108ba965d:21412:7)
```

**Comment by TuxedoTako** [comment]
I can't reproduce it with just hiding a random thread. Which board did this happen on? Which page? And which threads were hidden?

**Comment by saxamaphone69** [comment]
It popped up randomly for me on the index of /g/, page 3.

I tried refreshing and flipping through the pages again to try and get the error to pop up with no avail.

Reported it for the sake of reporting it. If it happens again and I can work out why, I'll share.

**Comment by saxamaphone69** [comment]
Hm, `76759434` is the sticky, which I had manually hidden (what feels like) a long time ago.

The `/sqt/` thread references the sticky. I (think) somewhere in that error stack it tried `fetchedPost`? Maybe something there?

**Comment by saxamaphone69** [comment]
Yep, that was it.

The other (native) archived thread in that same OP doesn't do the same error, just the CT link to the sticky.

https://github.com/TuxedoTako/4chan-xt/assets/877480/57c08c8a-6104-41b0-ab2c-dd7007ff6ce6

Interestingly, when you inline the backlink, it refers to the sticky as OP instead of CT - though this 'bug' is present in 4chan X too.

![image](https://github.com/TuxedoTako/4chan-xt/assets/877480/db11d37d-b0ac-485a-a25c-919fe4ee1dea)

And it seems when it does happen, when you head to page 10/11 to find the stubbed sticky, it finds itself unstubbed but anchored like the other stubs.

![image](https://github.com/TuxedoTako/4chan-xt/assets/877480/40209dc1-2d7d-47c0-884b-052922e1d97d)

Clicking the `-` to restub the thread does nothing, and you need to click it twice.

**Comment by TuxedoTako** [comment]
Should be fixed in [2.4.5](https://github.com/TuxedoTako/4chan-xt/releases).
